CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - Deployed V3 Balancer Contract Registry to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
 - Deployed V3 Aggregator Router to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
 - Deployed V3 MEV Capture Hook to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
+- Deployed V3 CowSwap Fee Burner to Mainnet, Gnosis, Arbitrum, Base and Sepolia.
```

Comment:
CowSwap or Cow Swap? Usually we've separated component words in the comments (though maybe CowSwap is a word; I don't feel strongly - just pointing it out in case it wasn't intentional).